app/assets/stylesheets/users.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
```

Comment:
please delete this empty file because don's use

Reply:
Done